docs/content/en/integrations/parsers/file/npm_audit.md

```diff
@@ -2,8 +2,29 @@
 title: "NPM Audit"
 toc_hide: true
 ---
+
+**Note: This parser only supports import from NPM Audit v6 or older.**
```

Comment:
https://github.com/DefectDojo/django-DefectDojo/issues/4285

Reply:
Could you rather resolve this issue?

Reply:
In case you have a v7 file, you can attach it to the linked issue and then I can help you advancing this parser.

Reply:
@manuel-sommer We've got a follow-on PR which will add a parser for v7 - should drop in the next day or two.

The data provided by npm_audit changed changed pretty significantly in v7 (to less useful data) so we're going to keep the parsers separate to:
* allow people to choose which version they want to run
* understand why there's a significant difference in output. If these were combined, you could have one parser producing 2 very different looking findings if results from v7 and earlier were uploaded
* a separate parser avoids a migration (a bonus given the above reasons)

Thanks for asking though :+1: